include/azure-tools.bash

```diff
@@ -1,11 +1,11 @@
 azure-configure-arm() {
     declare desc="Configure new ARM application"
-
-    docker run -it sequenceiq/azure-cli-tools:1.3 configure-arm "$@"
+    mkdir -p azure-cli-log
```

Comment:
-p      Create intermediate directories as required.  If this option is not specified, the full path prefix of each operand must
             already exist.  On the other hand, with this option specified, no error will be reported if a directory given as an operand
             already exists.  Intermediate directories are created with permission bits of rwxrwxrwx (0777) as modified by the current
             umask, plus write and search permission for the owner.